example/jupyter/tutorial_analyzer.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Analyzer](../../doc/analyzer_design.md) is designed to explain the machine learning model in SQLFlow. Here we are going to learn how to write an `ANALYZE` SQL.  \n",
+    "\n",
+    "We implement the analyzer based on [SHAP](https://github.com/slundberg/shap). By SQLFlow, an ANALYZE SQL will be translated to SHAP code. SQLFlow enables the code to read the dataset and load the trained model, then draws a figure to explain the model. At this stage, SQLFlow supports using the [TreeExplianer](https://github.com/slundberg/shap#tree-ensemble-example-with-treeexplainer-xgboostlightgbmcatboostscikit-learn-models) to draw a summary plot.\n",
+    "\n",
+    "## Syntax\n",
```

Comment:
Maybe move the syntax section to the user guide: https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/user_guide.md#syntax

Reply:
The syntax here just briefly describes how to use and introduces the `ANALYZER`.
Of course, it's a good idea to add a section about `analyzer` in the user guide.